### PR TITLE
Update names.ts

### PR DESCRIPTION
**Pull request by StunninglyWrong**
-fixed sorting (strictly alphabetical)
-tweaks to match in-game names
-add some missing names
-add missing items (ootBluePotion, ootGanonBossKey, mmGreenRupee)

### CODE REVIEW

File: lib/combo/names.ts [hunk 1]
```diff
@@ -3,20 +3,21 @@ const ITEM_NAMES: {[k: string]: string} = {
   MM_ARROW_ICE: "Ice Arrows (MM)",
   MM_ARROW_LIGHT: "Light Arrows (MM)",
   MM_ARROWS_30: "30 Arrows (MM)",
-  MM_BOMBCHU: "1 Bombchu (MM)",
-  MM_BOMBCHU_10: "10 Bombchu (MM)",
-  MM_BOMBCHU_5: "5 Bombchu (MM)",
-  MM_BOMBER_NOTEBOOK: "Bomber's Notebook",
   MM_BOMB_BAG: "Bomb Bag (MM)",
+  MM_BOMBCHU: "1 Bombchu (MM)",
+  MM_BOMBCHU_10: "10 Bombchus (MM)",
```
Comment (Nax) [inline lib/combo/names.ts:8]: The plural of `Bombchu` is just `Bombchu` (no `s`) according to in-game text.

**Comment by StunninglyWrong** [comment]
Let me preface this by saying I have very limited knowledge about all this. 
I noticed some Starting Items in the dev generator were missing their nice names, so I wanted to fix that. I ended up doing something more.
I wasn't 100% sure what you've decided to do with the bottles items' names, so I didn't go too deep into that. That said, I'm quite sure you wanna rename MM_BOTTLED_GOLD_DUST to MM_BOTTLE_GOLD_DUST everywhere.

File: lib/combo/names.ts (hunk 1 above)
Comment (StunninglyWrong) [reply lib/combo/names.ts:8]: Thanks for pointing that out, I see now that "Bombchus" as a plural was only introduced in more recent games. Back to Bombchu it is!

**Comment by Nax** [comment]
Merged on a branch as it need some minor changes (some names were changed on purpose, like MM tokens - so it fits the in-game screen), but otherwise good 👍

**Pull request merged**